targets/TARGET_NUVOTON/TARGET_M2351/device/system_M2351.h

```diff
@@ -130,19 +130,6 @@ extern void SystemInit(void);
 extern void SystemCoreClockUpdate(void);
 
 
-
-
-#if defined (__ICCARM__)
-uint32_t __TZ_get_PSP_NS(void);
-void __TZ_set_PSP_NS(uint32_t topOfProcStack);
-int32_t __TZ_get_MSP_NS(void);
-void __TZ_set_MSP_NS(uint32_t topOfMainStack);
-uint32_t __TZ_get_PRIMASK_NS(void);
-void __TZ_set_PRIMASK_NS(uint32_t priMask);
-#endif
-
-
-
```

Comment:
Why is this removed?

Reply:
they are dummy defines in code, may be old trial code

Reply:
I see, but it happens to be in target code which we are removing, hope its ok with Nuvoton.

Reply:
Actual PR for this change is https://github.com/ARMmbed/mbed-os/pull/9431
Please note: Commit for this PR is afb559b
Dependent on : #9431